conventions/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-shadowing.gradle.kts

```diff
@@ -18,7 +18,10 @@ tasks.withType<ShadowJar>().configureEach {
   relocate("java.util.logging.Logger", "io.opentelemetry.javaagent.bootstrap.PatchLogger")
 
   // prevents conflict with library instrumentation
-  relocate("io.opentelemetry.instrumentation", "io.opentelemetry.javaagent.shaded.instrumentation")
+  relocate("io.opentelemetry.instrumentation", "io.opentelemetry.javaagent.shaded.instrumentation") {
```

Comment:
The entire relocation definition is also duplicated in `io.opentelemetry.instrumentation.muzzle-check.gradle.kts`, can you update that file too?